File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Wind.java

package restaurant.sa.com.sarestaurant.appview.weather.model;

import com.google.gson.annotations.SerializedName;
public class Wind {

    @SerializedName("chill")
    private String mChill;
    @SerializedName("direction")
    private String mDirection;
    @SerializedName("speed")
    private String mSpeed;

    public String getChill() {
        return mChill;
    }

    public void setChill(String chill) {
        mChill = chill;
    }

    public String getDirection() {
        return mDirection;
    }

    public void setDirection(String direction) {
        mDirection = direction;
    }

    public String getSpeed() {
        return mSpeed;
    }

    public void setSpeed(String speed) {
        mSpeed = speed;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Wind.kt
package restaurant.sa.com.sarestaurant.appview.weather.model

import com.google.gson.annotations.SerializedName

class Wind {

    @SerializedName("chill")
    var chill: String? = null
    @SerializedName("direction")
    var direction: String? = null
    @SerializedName("speed")
    var speed: String? = null

}
